packages/preview/muw-touying/0.1.0/typst.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "muw-touying"
```

Comment:
It should be made clearer that this template is not an official one. The polylux template that you used as inspiration contains the word "community" in its name for example. Other common strategies to make the name unique enough so that it doesn't sound too official is to add an adjective or make an indirect reference to some University lore.